docs/static_site/src/_includes/get_started/windows/python/cpu/pip.md

```diff
@@ -70,4 +70,4 @@ $ pip install --pre mxnet -f https://dist.mxnet.io/python/all
 </div> <!-- End of master-->
 
 {% include /get_started/pip_snippet.md %}
-{% include /get_started/gpu_snippet.md %}
```

Comment:
what has changed in all these gpu_snippets?

Reply:
I bet it is eol character - those md files had no eol before

Reply:
Yes. My editor ensures each file has eol by default